STANDARDS.md

```diff
@@ -2,65 +2,69 @@
 This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-STANDARDS version: 3 (adopted 2020.12.11)
+STANDARDS version: 4 (adopted 2024.11.XX)
```

Comment:
Question: do we want to update the version number? Nothing has ever conformed to version 3 as far as I can tell from the submissions directory, so we could continue using it, but I thought it might be better to bump it so there is no confusion.

Reply:
SGTM!